contracts/test/strategies/aerodrome-amo.base.fork-test.js

```diff
@@ -46,6 +48,12 @@ describe("ForkTest: Aerodrome AMO Strategy empty pool setup (Base)", function ()
     await oethb
       .connect(rafael)
       .approve(aeroSwapRouter.address, oethUnits("1000"));
+
+    await deployWithConfirmation("AerodromeAMOQuoter", [
```

Comment:
this will fit better in here: https://github.com/OriginProtocol/origin-dollar/blob/f0d0ace54b15e64ff849c9d6b0169ddfb0fe5b19/contracts/test/_fixture-base.js#L4